### DISCUSSION TITLE
How to set scoped registry when deploying Next.js with Vercel?

### DISCUSSION THREAD

**Discussion by NathanSebhastian** [vercel/vercel#4736] [Help] [answered]
Hi, I'm trying to deploy a [Next.js app](https://github.com/nsebhastian/bit-next-components) to Vercel as a test drive, but I have an error when installing components from [Bit](https://bit.dev).

Upon further investigation, I think I need to set a scoped registry for `@bit` packages as shown in [this post](https://blog.bitsrc.io/introducing-bits-npm-package-registry-f4892de57b0c).

Is there a way to set scoped registry when deploying with Vercel? I've searched the documentation but can't find any.

Thanks!

**Comment by mcsdevv** [comment, marked as answer]
Hello @nsebhastian, does this article help at all?

- https://vercel.com/knowledge/using-private-dependencies-with-vercel

**Comment by NathanSebhastian** [comment]
@mcsdevv hi, yes I added NPM_RC environment variable for the registry from Vercel. Thanks!
